models/cycle_gan_model.py

```diff
@@ -261,12 +308,20 @@ def compute_G_loss(self):
             self.loss_G_B = self.criterionContrastive(-self.netD_B(self.real_A),-self.netD_B(self.fake_A))                                                    
         else:
             self.loss_G_B = self.criterionGAN(self.netD_B(self.fake_A), True)
+
+        if hasattr(self,"netD_A_2"):
+            self.loss_G_A_2 = self.criterionGAN(self.netD_A_2(self.fake_B), True)
```

Comment:
Could we do `self.loss_G_A += self.criterionGAN(self.netD_A_2(self.fake_B), True)` here instead ?

Reply:
We could but it will prevent us from displaying the loss from each discriminator.

Reply:
OK